### DISCUSSION TITLE
如何隐藏评论者的浏览器和操作系统信息？

### DISCUSSION THREAD

**Discussion by 2017zhangyuxuan** [walinejs/waline#576] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/43541108/134790263-739686b3-e55b-4716-903c-1f28f122b94e.png)
部署之后，进行评论，会携带评论者的浏览器和操作系统信息，如图所示。看了下官网文档，也没有相关参数来控制，想请教一下这个应该如何隐藏？

**Comment by lizheming** [comment]
可以自己写 CSS 控制隐藏
```css
.vmeta {
  display:none;
}
```

**Reply by xiaoao19** [reply]
您好，请问wordpress里怎么加入这个参数，
我目前的代码是：
`
<div id="waline"></div>
<script>
Waline.init({
el: '#waline',
lang: 'en',
copyright: false,
serverURL: 'Vercel_URL',
});
</script>

`

**Reply by xiaoao19** [reply]
chatgpt让我这样加但是没有用：

您可以使用以下代码来将 .vmeta{display:none;} 加入到您的代码中：

Copy code
<style>
  .vmeta{
    display:none;
  }
</style>

<script src="https://unpkg.com/@waline/client@v2/dist/waline.js"></script>
<link rel="stylesheet" target="_blank" href="https://unpkg.com/@waline/client@v2/dist/waline.css">
这样，您的样式就会被正确地应用到您的网站上。

请注意，您可能需要使用不同的方式将样式添加到网站中，具体取决于您使用的框架和构建流程。

**Comment by lizheming** [comment, marked as answer]
更新，服务端有个 `DISABLE_USERAGENT` 的配置用于不展示用户设备信息 https://waline.js.org/reference/server.html#%E7%8E%AF%E5%A2%83%E5%8F%98%E9%87%8F

**Reply by 2017zhangyuxuan** [reply]
在vercel重新配置后好像并没有生效。。

**Reply by 2017zhangyuxuan** [reply]
在vercel生效了，但不知道为什么hexo配置waline，本地预览还是会显示

**Reply by lizheming** [reply]
看了下代码应该不会，如果会的话可以把地址发下我之后去复现下看看。

**Reply by 2017zhangyuxuan** [reply]
感谢回复！不好意思，是我重新部署后，忘记更新server地址了，现在没有问题了
